src/main/java/org/opentripplanner/gtfs/mapping/LinkStopsAndParentStationsTogether.java

```diff
@@ -28,6 +32,12 @@ void link(Collection<org.onebusaway.gtfs.model.Stop> gtfsStops) {
                 Stop otpStop = getOtpStop(gtfsStop);
                 Station otpStation = getOtpParentStation(gtfsStop);
 
+                if (otpStation == null) {
+                    LOG.warn("Parent station {} not found. Stop {} will not be linked to a parent"
+                            + " station.", gtfsStop.getParentStation(), gtfsStop.getId().getId());
+                    continue;
```

Comment:
This should probably be a GraphBuilderAnnotation ?

Reply:
Agreed, the typical pattern for this is logging the result of adding a graph builder annotation. Other than that, the change looks fine.

Reply:
I have changed the code to add a graph builder annotation instead of writing a log message. We also have a cleanup of the annotation/logging during graph build in our branch that we plan to port to OTP2.

Reply:
I added a new issue related to this yesterday: Improve OTP data import report #2869